algorithms.py

```diff
@@ -642,6 +657,9 @@ def hudson_simulate(self):
                 elif min_time == t_ca:
                     # print("CA EVENT")
                     self.common_ancestor_event(ca_population, 0)
+                elif min_time == t_cen:
```

Comment:
We don't want to do it this way --- we want to register the census event as one of the ``modifier_events`` and use the general infrastructure for dispatching these events. We shouldn't be touching the main loop of the simulation here at all --- all we want is

1) a function that runs the census event (modeled after a population parameters change, say)
2) We an event referencing this function to the ``modifier_events`` list so that this function gets executed at the correct time.